config/symbols.pspeu.stwrp.txt

```diff
@@ -71,9 +71,9 @@ EntityWarpRoom = 0x09244d28;
 EntityWarpSmallRocks = 0x09245c50;
 g_eRedDoorUV = 0x09246080;
 UNK_Invincibility0 = 0x09246098;
-D_801810A0 = 0x092460C8;
-D_801810B0 = 0x092460D8;
-D_80181110 = 0x09246138;
+g_ESoulStealOrbAngles = 0x092460C8;
+g_ESouldStelOrbSprt = 0x092460D8;
```

Comment:
Is this spelling intentional? Sould / Stel

Reply:
Done this and and the others, nice catch!